gatsby-browser.js

```diff
@@ -31,6 +31,10 @@ export const wrapPageElement = ({ element }) => (
 
 export const onRouteUpdate = ({ location }) => scrollToAnchor(location)
 
+export const onServiceWorkerUpdateFound = () => {
+  window.location.reload(true)
+}
+
```

Comment:
I would not go this direction, it could potentially cause very strange behaviour.
Better to isolate the problem and solve it. I've added some comments to notion.

Reply:
What about:

```
if (window.confirm('This app has updated, please reload the page')) {
  window.location.reload(true)
}
```

So it leaves a choice to the user, to:
- reload or not the page (if `okay` is clicked at confirmed) and get the updates
- alerts them that some updates are available, and that some strange behavior might happens if they don't reaload (if `no` is clicked at confirm).